angular.json

```diff
@@ -32,12 +32,12 @@
             "aot": true,
             "assets": ["src/assets"],
             "scripts": [],
-            "styles": ["src/styles/themes/default/style.scss"],
+            "styles": ["src/styles/themes/b2b/style.scss"],
```

Comment:
This must also become `placeholder` to be properly replaced. For this you also have to create the file to satisfy Angular configuration validation.

Reply:
Let me try something 😃

Reply:
I had changed this too but then the build results in an error.

Reply:
Angular config validation expects this file to exist and it is run before the webpack.custom replaces it. So as long as the replacing is not done by an Angular builder, it has to be that way. 😅 

see 61b006e